Resources/public/js/views/ez-testoneview.js

```diff
@@ -0,0 +1,298 @@
+        _showIntro: function() {
+            var selected = this.get('selected'),
+                introContainer = this.get('container').one('#ez-testone-intro');
+            if (selected) {
+                introContainer.hide();
+                this.get('container').addClass('ez-testone-loading');
+                this.get('service').loadIntro(selected.locationId);
+            }
+        },
+
+        /**
+         * Listener for loaded intro
+         *
+         * @param {Y.eZ.TestOneView} view
+         * @returns {Function}
+         * @private
+         */
+        _onIntroLoaded: function(view) {
+            return function(e) {
+                var selected = view.get('selected'),
+                    introContainer = view.get('container').one('#ez-testone-intro');
+
+                view.get('container').removeClass('ez-testone-loading');
+                introContainer.show();
+                view.get('introView').set('intro', {
+                    field: e.intro,
+                    name: selected.name
+                });
+            };
+        },
+
+        /**
+         * Listener for failed load of intro
+         *
+         * @param {Y.eZ.TestOneView} view
+         * @returns {Function}
+         * @private
+         */
+        _onIntroLoadFailed: function(view) {
+            return function(e) {
+                var introContainer = view.get('container').one('#ez-testone-intro');
+
+                view.get('container').removeClass('ez-testone-loading');
+                introContainer.show();
+            };
+        },
+
+        /**
+         * Loads article list
+         *
+         * @method _loadArticleList
+         * @protected
+         */
+        _loadArticleList: function() {
+            var service = this.get('service'),
+                that = this,
+                current = this.get('current'),
+                selected = this.get('selected'),
+                loader = Y.Node.create('<span class="ez-inline-loader"></span>'),
+                articlesContainer = this.get('container').one('#ez-testone-articles');
+
+            if (!selected) {
+                articlesContainer.append(loader);
+            } else {
+                selected.target.append(loader);
+            }
+            articlesContainer.addClass('ez-testone-loading');
+
+            service.loadArticleList(current.locationId, function(articles) {
+                var articleMap = {};
+                Y.Array.each(articles, function(article) {
+                    articleMap[article.locationId] = article;
+                });
+                loader.remove();
+                articlesContainer.removeClass('ez-testone-loading');
+                that.get('articlesView').set('articles', articleMap);
+            });
+        },
+
+        /**
+         * Destructor
+         */
+        destructor: function () {
+            this.get('articlesView').removeTarget(this);
+            this.get('articlesView').destroy();
```

Comment:
Tutaj zrobiłbym to tak: `this.get('articlesView').destroy({remove: true});`. Dzięki temu, kod HTML też zostanie usunięty.